src/KOKKOS/pair_reaxff_kokkos.h

```diff
@@ -81,25 +85,25 @@ template<int NEIGHFLAG>
 struct PairReaxUpdateBond{};
 
 template<int NEIGHFLAG, int EVFLAG>
-struct PairReaxComputeBond1{};
+struct PairReaxFFomputeBond1{};
 
 template<int NEIGHFLAG, int EVFLAG>
-struct PairReaxComputeBond2{};
+struct PairReaxFFomputeBond2{};
 
 template<int NEIGHFLAG, int EVFLAG>
-struct PairReaxComputeMulti1{};
+struct PairReaxFFomputeMulti1{};
 
 template<int NEIGHFLAG, int EVFLAG>
-struct PairReaxComputeMulti2{};
+struct PairReaxFFomputeMulti2{};
```

Comment:
Find/replace mistake here and some other places, should be `PairReaxFFCompute*`

Reply:
well spotted. thanks.